Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:0.10.36
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app/
+RUN npm install
+COPY . /usr/src/app
+
+# replace this with your application's default port
+EXPOSE 8080 
+
+CMD [ "npm", "start" ]
```

Comment:
This will still run as root. You'll need to create a new user with some variant of `useradd` and include a `USER` command to run it as a different user, and make sure that everything has the correct permissions.